### PR TITLE
pushing eevee intent metrics to db.

### PR DESCRIPTION
**Pull request by greed2411**
`irr_from_tog` now has `mlwr=True` pipeline arg (which also requires  `slu_project_name`) for pushing calculated eevee metrics on a tog job to ml-metrics db, intent_metrics table.

### CODE REVIEW

File: skit_pipelines/components/push_irr_to_postgres.py [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+    import pandas as pd
+    import psycopg2
+    from loguru import logger
+
+
+    ML_METRICS_DB_NAME = os.environ["ML_METRICS_DB_NAME"]
+    ML_METRICS_DB_HOST = os.environ["ML_METRICS_DB_HOST"]
+    ML_METRICS_DB_PORT = os.environ["ML_METRICS_DB_PORT"]
+    ML_METRICS_DB_USER = os.environ["ML_METRICS_DB_USER"]
+    ML_METRICS_DB_PASSWORD = os.environ["ML_METRICS_DB_PASSWORD"]
```
Comment (biswaroop1547) [inline skit_pipelines/components/push_irr_to_postgres.py:27]: can we define these under constants.py and then call it?

File: skit_pipelines/components/push_irr_to_postgres.py [hunk 2]
```diff
@@ -0,0 +1,122 @@
...
+    INSERT_SQL_QUERY = """
+    INSERT INTO project_event 
+    (name, reference_id, support, precision, recall, f1, 
+        raw, is_deleted, is_complete, created_at, updated_at, 
+        of_type, language, app_id, created_date)
+    VALUES 
+    (%(name)s, %(reference_id)s, %(support)s, %(precision)s, %(recall)s, %(f1)s, 
+        %(raw)s, %(is_deleted)s, %(is_complete)s, %(created_at)s, %(updated_at)s, 
+        %(of_type)s, %(language)s, %(app_id)s, %(created_date)s);
+    """
```
Comment (biswaroop1547) [inline skit_pipelines/components/push_irr_to_postgres.py:39]: should go under constants.py too

File: skit_pipelines/components/push_irr_to_postgres.py [hunk 3]
```diff
@@ -0,0 +1,122 @@
...
+    dataset_job_id = 3091
+    language = "en"
+    slu_project_name = "indigo"
+    
+    a = push_irr_to_postgres(
+        eevee_intent_df_path,
+        dataset_job_id,
+        language,
+        slu_project_name
+    )
```
Comment (biswaroop1547) [inline skit_pipelines/components/push_irr_to_postgres.py:122]: can we remove the example from here and add it as a section under irr_from_tog docstring on how to get this behaviour through charon?

**Review by biswaroop1547 — CHANGES_REQUESTED** [review]

File: skit_pipelines/components/push_irr_to_postgres.py (hunk 3 above)
Comment (greed2411) [reply skit_pipelines/components/push_irr_to_postgres.py:122]: this helps with running things locally. i can comment it out  if you want.

Comment (biswaroop1547) [reply skit_pipelines/components/push_irr_to_postgres.py:122]: yep okay, would be helpful if you could add a section on this in docs with example values

File: skit_pipelines/components/push_irr_to_postgres.py (hunk 1 above)
Comment (greed2411) [reply skit_pipelines/components/push_irr_to_postgres.py]: this already existed.

File: skit_pipelines/components/push_irr_to_postgres.py (hunk 2 above)
Comment (greed2411) [reply skit_pipelines/components/push_irr_to_postgres.py]: moved it out there

**Comment by greed2411** [comment]
found an issue, where all the irr-metrics/events are getting pushed to ingio's name. (because i thought app_id wouldn't matter). but it actually matters. want to explore if we are okay with creating a new table for this.

File: skit_pipelines/components/push_irr_to_postgres.py (hunk 3 above)
Comment (greed2411) [reply skit_pipelines/components/push_irr_to_postgres.py]: done. added docstring containing mlwr=True example.

**Comment by greed2411** [comment]
the PR is good for merging and it is working as expected. please go ahead with the pull request.

**Comment by greed2411** [comment]
have written bit of code, introduced a component to extract data out of the tagged content etc to populate the intent_metrics table.
now it is ready for review. sorry for the to & fro.

**Comment by greed2411** [comment]
resolved merge conflicts.

**Comment by greed2411** [comment]
`raw` column as messed up sorry!

**Comment by greed2411** [comment]
had to fix the `raw` column output. now using the json dump of the classification_report, we can rebuild it on metabase.
example looking metabase question: https://metabase.skit.ai/question/4838-classification-report-intent-metrics?intent_metrics_id=11
for the run: https://kubeflow.skit.ai/_/pipeline/?ns=skit#/runs/details/5b946f60-4800-42a1-8607-6cdd01ee308f

please have a look at it and give me feedback. this is the final PR review request for real.

**Review by biswaroop1547 — APPROVED** [review]

**Pull request merged**